ncard_app/models.py

```diff
@@ -23,16 +23,17 @@ class OrganisationType(models.IntegerChoices):
 
     name = models.CharField('name', max_length=255)
     primary_contact = models.ForeignKey('ncard_app.Person', on_delete=models.RESTRICT, null=True, blank=True, related_name='organisations_primary_contact')
-    phone = models.CharField('phone', max_length=25, blank=True, validators=[phone_validator])
-    website = models.URLField('website', blank=True)
-    twitter_handle = models.CharField('Twitter handle', max_length=16, blank=True, validators=[twitter_validator])
+    phone = models.CharField('phone', max_length=25, blank=True, null=True, validators=[phone_validator])
+    website = models.URLField('website', blank=True, null=True)
+    twitter_handle = models.CharField('Twitter handle', max_length=16, blank=True, null=True, validators=[twitter_validator])
```

Comment:
Don't use null=True for CharFields or URLFields. Django stores blank text as '' (the empty string).